StreetCare/StreetCare/Views/Screens/ICanHelpView.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  ICanHelpView.swift
+//  StreetCare
+//
+//  Created by Amey Kanunje on 9/27/24.
+//
+
+import SwiftUI
+
+struct ICanHelpView: View {
+    
+    @Binding var isPresented: Bool
+    @State private var isOutreachCreated = false
+    
+    var body: some View {
+        VStack(alignment: .leading, spacing: 20) {
+                    HStack {
+                        Spacer()
+                        Button(action: { isPresented = false }) {
+                            Image(systemName: "xmark")
+                                .foregroundColor(.gray)
+                        }
+                    }
+                    
+                    Text("Make sure you are not going alone")
```

Comment:
There are four text views here could you abstract this into a function instead of repeating the same code?

Reply:
> There are four text views here could you abstract this into a function instead of repeating the same code?

Resolved